src/routes/docs/advanced/platform/fair-use-policy/+page.markdoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: article
+title: Fair use policy
+description: Understand Appwrite's usage limits, prohibited activities, and enforcement actions.
+---
+
+At Appwrite, we are committed to providing high-quality, reliable, and scalable backend services for all users. Our Fair Use Policy ensures that resources are used responsibly and that every user receives a consistent experience. This policy applies to all users and outlines acceptable usage patterns and limitations.
+
+# Definitions and scope {% #definitions-and-scope %}
+
+- **Normal usage:** Resource usage that falls within expected thresholds for a user’s selected plan.
+- **Excessive usage:** Usage that exceeds defined thresholds and may affect the platform’s performance for other users.
+- **Prohibited activities:** Actions or behaviors that violate this policy, including, but not limited to, abuse of resources and security breaches.
+
+This policy applies to all resources provided by Appwrite Cloud, including but not limited to API requests, data storage, compute resources, and database operations.
+
+# Usage limits and thresholds {% #usage-limits-and-thresholds %}
+
+Each user plan includes specific resource limits, which are available to view on our [Pricing page](/pricing).
```

Comment:
I'm really not sure about putting usage limits on this page since we already have that kind of info on pages like https://appwrite.io/docs/advanced/platform/pro#reaching-resource-limits

Reply:
Vercel also retained this info in their fair use policy, which is what inspired this particular subsection

Reply:
ok we'll just leave it in there for now.